rust/src/lib/nispor/ethtool.rs

```diff
@@ -20,8 +81,24 @@ fn gen_ethtool_config(ethtool_info: &nispor::EthtoolInfo) -> EthtoolConfig {
         pause_config.autoneg = Some(pause.auto_negotiate);
         ret.pause = Some(pause_config);
     }
+    // TODO(Gris Ge): Use `OnceCell`
+    let supported_features: HashSet<&&str> =
+        SUPPORTED_ETHTOOL_FEATURES.iter().collect();
```

Comment:
I think you can avoid the `&&` using `iter().copied()` or `cloned()`?

Reply:
`&&` is better than clone action.
Never mind, let me fix it for good with OnceCell.